### PR TITLE
Singledomain preview

### PR DESCRIPTION
**Pull request by variousred**
fix for showing previews in cms

### CODE REVIEW

**Comment by variousred** [comment]
@bbauer https://www.pivotaltracker.com/story/show/85356354

File: app/models/web_home_template.rb [hunk 1]
```diff
@@ -27,3 +27,4 @@ def relative_path
     "/"
   end
 end
+
```
Comment (bbauer) [inline app/models/web_home_template.rb]: Has anyone really ever had to cat two files together, in the history of G5?

**Comment by bbauer** [comment]
:+1:

File: app/models/web_home_template.rb (hunk 1 above)
Comment (variousred) [inline app/models/web_home_template.rb]: I LIKE THE WAY IT LOOKS IN VIM!!!!!

Comment (bbauer) [inline app/models/web_home_template.rb]: FINE!

Comment (bbauer) [inline app/models/web_home_template.rb]: :) Honestly I'm fine either way...

Comment (variousred) [inline app/models/web_home_template.rb]: :D

**Pull request merged**